examples/bin/supervise

```diff
@@ -179,13 +195,23 @@ usage() unless GetOptions(
   'vardir|d=s',
   'kill-timeout|t=i',
   'chdir=s',
-  'svlogd:s'
+  'svlogd:s',
+  'command=s@'
 );
 
-usage() unless $opt{'conf'} && $opt{'vardir'};
+usage() unless ((@{$opt{'command'}} || $opt{'conf'}) && $opt{'vardir'});
```

Comment:
I don't know enough perl but something in this line is messed up

When I run

```
./bin/start-micro-quickstart 
Can't use an undefined value as an ARRAY reference at /Users/vadim/Projects/apache-druid/distribution/target/apache-druid-26.0.0-SNAPSHOT/bin/supervise line 202.
```

Reply:
Fixed.